src/quo2/foundations/typography.cljs

```diff
@@ -1,25 +1,40 @@
 (ns quo2.foundations.typography)
 
-;; Formulat for letter spacing from figma %: font-size*{figma-percentage}/100
+(defn- tracking-fn
+  "Check inter dynamic metric https://rsms.me/inter/dynmetrics/"
+  [font-size]
+  (let [a -0.0223
+        b 0.185
+        c -0.1745
+        e js/Math.E]
+    (->> font-size
+         (* c)
+         (js/Math.pow e)
+         (* b)
+         (+ a)
+         (* font-size))))
```

Comment:
why `(* font size)? I don't see this in formula.

Reply:
The formula you get is in unit `em` which is `-0.0031585560420509633` for `13px`, rounded to `0.3%` in Figma.
So `-0.0031585560420509633em` for `13px` which is why there's a `(* font-size)`